task/sonarqube-scanner/0.1/sonarqube-scanner.yaml

```diff
@@ -28,8 +28,19 @@ spec:
     - name: SONAR_PROJECT_KEY
       description: Project's unique key
       default: ""
+    - name: SONAR_LOGIN_SECRET_NAME
+      description: Sonar login secret name
+      default: ""
+    - name: SONAR_LOGIN_SECRET_KEY
+      description: Sonar login secret key
```

Comment:
I guess this param should also have a default value. WDYT?

Reply:
added default value